src/components/map/layers/NetworkLayers.tsx

```diff
@@ -28,13 +22,15 @@ interface INetworkLayersProps {
     networkStore?: NetworkStore;
 }
 
+function getGeoServerUrl(layerName: string) {
+    const GEOSERVER_URL = process.env.GEOSERVER_URL || 'http://localhost:8080/geoserver';
+    // tslint:disable-next-line:max-line-length
+    return `${GEOSERVER_URL}/gwc/service/tms/1.0.0/joremapui%3A${layerName}@EPSG%3A900913@pbf/{z}/{x}/{y}.pbf`;
```

Comment:
You forgot to include this very important comment :P
// TODO: Don't hardcode	
// Will be fixed in: https://github.com/HSLdevcom/jore-map-ui/issues/235

Reply:
Well, this is #235, I didn't really think it was hardcoded anymore. I guess it is somewhat hardcoded, but '/gwc/service/tms/1.0.0/joremapui%3A${layerName}@EPSG%3A900913@pbf' is just declaring the type of tiles we want. This functionality is connected to other things in this file, for example that we specify we read the tiles in tms format

Reply:
Ah yes right, I didnt notice this was #235 :P I agree it is less hard coded now. Part of it still is, but maybe it is not worth fixing to a separate file